src/fontra/client/lang/en.json

```
@@ -73,6 +73,8 @@
   "editor.hand-tool": "Hand Tool",
   "editor.knife-tool": "Knife Tool",
   "editor.pen-tool": "Pen Tool",
+  "editor.pen-tool-cubic": "Pen Tool (bezier)",
```

```suggestion
  "editor.pen-tool-cubic": "Pen Tool (cubic)",
```
(quad curves are also Bezier, just not cubics)